contracts/Gauge.sol

```diff
@@ -324,7 +497,7 @@ contract Gauge is IGauge {
         if (reward > 0) {
             rewards[_account] = 0;
             if (_lock) {
```

Comment:
might want to add a comment here that _lock can only be true on msg.sender calls in this implementation, so no one tries to do this with a passed in account in the future.  At first glance it looks concerned that the else transfers to an account, but the if deposits for msg.sender, a different actor.